src/actions/router.js

```diff
@@ -61,7 +61,7 @@ export function goToChannel(channelOrChannelId, options) {
       // Assume we don't have always have all channels in the future.
       if (!channel) channel = {id: channelOrChannelId, slug: ''}
     }
-    const slug = channel.slug == null ? channel.mate.username : channel.slug
+    const slug = channel.slug == null ? channel.partner.username : channel.slug
```

Comment:
hope you looked over the entire code based for "mate" and replaced it with "partner". Also such things are better done in a separate PR.